### ISSUE TITLE
Installer raises windows 10 protection pop-up window

### ISSUE THREAD

**Issue by mjhalwa** [OpenCppCoverage/OpenCppCoverage#138]
When running the Installer `OpenCppCoverageSetup-x64-0.9.9.0.exe` on my Windows 10 Pro machine (Version 2004) windows prompts the following window (german Windows)

![grafik](https://user-images.githubusercontent.com/8994014/104306718-8ff31200-54ce-11eb-9082-be59f00f280f.png)

This is a screenshot I found on the web representing the same window in english:

![english alike screenshot](https://www.ghacks.net/wp-content/uploads/2012/11/windows-10-protected-your-pc.png)

I don't think this is a problem solely related to a missing signature - which most programs on the web have, because I never saw that window before (e.g. installing [WinMerge v2.16.6](https://winmerge.org/downloads/?lang=de)). I guess it is either some meta-information missing or an issue of the installer-framework you are using.

**Comment by uroni** [comment]
I think the first few people trying to run it will get this dialog. After enough people have clicked "Run anyway" it gets enough reputation to get a less scary dialog.

But having the installer signed would be great and would make this problem go away.

**Comment by GitMensch** [comment]
Yes, this is about the "reputation", which the setup file now has, so you won't see this.
I suggest to close this, because "sign installer and executables" is a different thing (and, until someone provides a link for free code-signing certificates, unlikely to come).

**Comment by OpenCppCoverage** [comment]
Thank you for reporting this issue and for contributing to the project.

The project is now being formally retired and the repository will be archived. I am therefore closing all remaining issues, as no further investigation, fixes, or releases are planned.

This administrative closure does not mean that the issue was resolved or that the report was invalid. The discussion will remain available for anyone who wants to address it in a fork.

Thank you for your understanding.
